scripts/generate-integration-pages.ts

```diff
@@ -170,7 +170,11 @@ function absoluteLinks({ base }: { base: string }) {
 		function visitor(node: Link | Definition) {
 			// Sanitize URL by removing leading `/`
 			const relativeUrl = node.url.replace(/^.?\//, '');
-			node.url = new URL(relativeUrl, base).href;
+			// Don't add absolute path to local and docs links.
+			node.url =
+				node.url.startsWith('#') || node.url.startsWith('/')
+					? node.url
+					: new URL(relativeUrl, base).href;
```

Comment:
I think we only want to do this for `#` fragment links. Links starting with `/` are relative to `https://github.com/` in the README, so should be made absolute.

There do appear to be some bare links to docs in the MDX integration README, but these should be updated to use absolute links as those links currently won’t work for someone reading the README on GitHub or npm.

```suggestion
			node.url = node.url.startsWith('#') ? node.url : new URL(relativeUrl, base).href;
```

Will try to mop the various issues up:

- [ ] Update the README at source in the astro repo
- [ ] Update this PR with this suggestion & re-run
- [ ] Fix the anchor links that are incorrect currently:
  - [ ] astro PR for the error page
  - [ ] docs PR for translations